### ISSUE TITLE
Error in callback for watcher "config": "TypeError: Cannot read property 'call' of undefined"

### ISSUE THREAD

**Issue by holic-cl** [vue-bulma/datepicker#46]
Hi, i am getting this error when i try to set the config param to mode: 'range'.

```
<datepicker placeholder="Select a range" :config="{ mode: 'range' }"></datepicker>
```

Any suggestion would be helpful.

Regards.

**Comment by jaybeecave** [comment]
seems to happen when config is set, looking at this now

**Comment by jaybeecave** [comment]
PR #39 will solve this if its merged.

**Comment by holic-cl** [comment]
Thanks, i will wait for that.

**Comment by luventa** [comment]
merged pr #38 which has the same code change as #39.
This issue shall be fixed. I'm upgrading the version now.

**Comment by luventa** [comment]
new version published. Please check if this issue is still there or not.
Feel free to reopen this issue. Thanks!
